test/Microsoft.IdentityModel.TestUtils/ExpectedException.cs

```diff
@@ -208,7 +208,9 @@ public static ExpectedException SecurityTokenEncryptionKeyNotFoundException(stri
 
         public static ExpectedException SecurityTokenUnableToValidateException(string substringExpected = null, Type innerTypeExpected = null)
         {
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
>CS0618 [](http://example.com/codeflow?start=24&length=6)

I am curious which member on this line is marked as obsolete? is it SecurityTokenUnableToValidateException?